### ISSUE TITLE
Tab Index Missing

### ISSUE THREAD

**Issue by ZafarSaeedKhan** [mseemann/angular2-mdl#288]
Global Tab index attribute is missing.
Can I provide a solution for it or can I propose a solution for it?

**Comment by mseemann** [comment]
hi. what do you mean by Global Tab index?

**Comment by ZafarSaeedKhan** [comment]
HTML tabindex Attribute:
Number that Specifies the tabbing order of the element (1 is first)
http://www.w3schools.com/TAgs/att_global_tabindex.asp

**Comment by mseemann** [comment]
ah! I understand. Yes, if you want you can create a PR for this.

**Comment by ulic75** [comment]
If someone wants to use tabIndex then it should be optionally supported.  Using a tabIndex > 0 anywhere on the page means that you have to use it everywhere on the page if you really want it work.

Here is an article for reference http://adrianroselli.com/2014/11/dont-use-tabindex-greater-than-0.html

I also created a [plunker](http://plnkr.co/edit/0mpe2kN9uK6rxe3vNWpL?p=preview) (based off one in the readme) that demonstrates the issue.  To see it in action run the plunker, click on the username text field, then press tab a few times.  You'll see that it tabs Username -> Password -> Link -> Login.  Probably not the desired result.

This happens because the two text fields have an equal tabIndex of 1, because those are the only two items with a defined index then the browser cycles through everything else.  I've gone down the tabIndex road before, it's almost never the right answer to the problem.

So to recap, IMO tabIndex should not be added to any element unless it's specifically added.  Currently it appears to be added to radios, checkboxes, and textfields automatically.

**Comment by mseemann** [comment]
@ZafarSaeedKhan can you check this. thx!

**Comment by mseemann** [comment]
@ulic75 you are right. the tabindex defaults to 1 - i'll change this to null.
